.github/workflows/node-sdk-schema-integrate.yml

```diff
@@ -0,0 +1,35 @@
+name: "Node: @serverless/sdk-schema: Integrate"
```

Comment:
I'm fine to follow this convention, but looking at our actions, we do not do this for any of them. I just followed what existed in the repo. So we should change them all then.

Reply:
Actually, _Integrate_ and _Validate_ should be setup directly in workflows we have currently.

See there are already two different npm packages configured in a way that jobs are run only on those which were changes (this commit landed today, so be sure to sync). So ideally `sdk-schema` should be added there as third

Reply:
Still, for `proto` linting,  it's best to introduce a separate workflow (it can be one file that runs on both PRs and merges to `main`) - at least _Integrate_ and _Validate_ are naturally restricted to operate only on `node` context and are very node specific